target/linux/bcm63xx/base-files/etc/board.d/01_leds

```diff
@@ -25,7 +25,8 @@ adb,pdg-a4001n-a-000-1a1-ax)
 	ucidef_set_led_netdev "lan" "LAN" "$model:green:internet" "eth0.1"
 	ucidef_set_led_netdev "wan" "WAN" "$model:green:adsl" "eth0.2"
 	ucidef_set_led_netdev "wlan0" "WIFI" "$model:green:wifi" "wlan0"
-	ucidef_set_led_usbdev "usb" "USB" "$model:green:service" "1-1"
+	ucidef_set_led_usbdev "usb1" "USB1" "$model:green:service" "1-1"
+	ucidef_set_led_usbdev "usb2" "USB2" "$model:red:service" "2-1"
```

Comment:
Does this handling of green and red LEDs match vendor behavior?

Reply:
This is the vendor manual:

Reply:
Yes, it does